src/App.css

```diff
@@ -402,6 +402,12 @@ main p {
 
 .app__header {
   padding: 20px;
+  display: flex;
```

Comment:
Also, please could you remove the code you've added for logo centering, I've fixed it in my PR.

Reply:
Thanks. I have updated the changes.